pl.lproj/Localizable.strings

```diff
@@ -82,18 +82,18 @@
 
 "COLLECTION_VIEW_EMPTY_SECTION_TITLE" = "Brak Stacji";
 
-"WELCOME_COPY" = "Witamy w Broadcasts! Dodaliśmy już parę naszych Irlandzkich stacji radiowych, aby twoja biblioteka nie była pusta.\n\nJeżeli wolisz, możesz dodać ręcznie własne stacje lub wybrać którąś z tysiąca stacji w naszym katalogu.";
+"WELCOME_COPY" = "Witamy w Broadcasts! Dodaliśmy już kilka irlandzkich stacji radiowych, aby Twoja biblioteka nie była pusta.\n\nJeżeli wolisz, możesz dodać ręcznie własne stacje lub wybrać którąś z tysiąca stacji z naszego katalogu.";
```

Comment:
`"WELCOME_COPY" = "Witamy w Broadcasts! Dodaliśmy już kilka irlandzkich stacji radiowych, aby Twoja biblioteka nie była pusta.\n\nJeżeli wolisz, możesz dodać ręcznie własne stacje lub wybrać którąś z tysiąca stacji z naszego katalogu.";`

→ 

`"WELCOME_COPY" = "Witamy w Broadcasts! Dodaliśmy już kilka irlandzkich stacji radiowych, aby Twoja biblioteka nie była pusta.\n\nJeżeli wolisz, możesz ręcznie dodawać własne stacje lub wybrać którąś z tysięcy stacji z naszego katalogu.";`

Tysiąca stacji means "1 thousand stations" which is technically incorrect, at least compared to you EN copy.